### DISCUSSION TITLE
How do I unstretch the video?

### DISCUSSION THREAD

**Discussion by poppykyu** [aniyomiorg/aniyomi#983] [Q&A] [answered]
I've recently updated my Aniyomi from an old version, suddenly, the videos when I watch anime are stretched horizontally. Before I updated, the ratio of the vids are normally at 16:9 with empty black spaces on the sides, now the video covers the entire screen of my phone. How do I unstretch the video?

**Comment by 002UwU-fan** [comment, marked as answer]
![Screenshot_2023-05-16-04-59-57-38_423fe07c33ed17f15a4e92e5e3a15972.jpg](https://github.com/jmir1/aniyomi/assets/107838391/25dd63f6-c0f2-4660-b36b-dd95854090b7)

Click on this button and it will change the aspect ratio

**Reply by jzrxl** [reply]
how do i change the media player like that?

**Reply by 002UwU-fan** [reply]
Download the preview version r6147

The link :
https://aniyomi.org/download/

Note: actually the preview version is more stable than the stable version 😂
